src/module/helpers/actor.ts

```diff
@@ -130,8 +131,8 @@ export function clicker_stat_card(
 ): string {
   let button = "";
   // Determine whether this is an unlinked token, so we can encode the correct id for the macro.
-  const r_actor = options.data.root.actor;
-  let id = r_actor.token && !r_actor.token.isLinked ? r_actor.token.id : r_actor.actor.id!;
+  const r_actor = options.data.root.actor as LancerActor;
+  let id = r_actor.token && !r_actor.token.isLinked ? r_actor.token.id : r_actor.id!;
```

Comment:
I assume this is for an unrelated bugfix?

Reply:
It was breaking linked sheets for me, so I had fix this just to get it tested.